source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -0,0 +1,44 @@
+.. _compass-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Querying with natural language in Compass is powered by Generative AI
+(Gen AI), and may give inaccurate responses. Please see our `Generative AI FAQ 
+<https://dochub.mongodb.org/core/faq-ai-features>`__ 
```

Comment:
(question)
You wouldn't use the dochub link in our docs, right? Instead you'd link to where ever the dochub link is directing, no?

Reply:
This one is a little tricky, in terms of when each of these docs is published. I am not 100% on the timeline, so thats why I choose to use the dochub link here. Knowing that I could easily change the link when the docs were fully published. 

I think if we can coordinate the merge order as:
1. Relational Migrator
2. Compass 
3. Charts
4. Docs landing

Then I can update the link, I have seen a little usage of doc-hub links in our docs:

![Screenshot 2024-02-23 at 12 06 56 PM](https://github.com/mongodb/docs-compass/assets/85948430/9bf2ffd8-4167-41f0-9811-c4eec6f72dd0)

An added bonus would be that the 404 nightly checker could give us a heads up if any doc-hub links are 404ing. @sarah-olson-mongodb